packages/aws-lambda-otel-extension/opt/otel-extension/external/custom-http-listener.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const http = require('http');
+const { logMessage } = require('../lib/helper');
+const { SAVE_FILE } = require('./helper');
+const { writeFileSync } = require('fs');
+
+// mainEventData is assigned in this file and then used in the parent file
+// eslint-disable-next-line no-unused-vars
+function customListen({ port, logsQueue, mainEventData, callback, liveLogCallback }) {
+  // init HTTP server for the Logs API subscription
```

Comment:
I'd remove this comment, and instead make `customListen` variable name self explanatory.

How about `initializeLogsApiListenerServer` ?